src/libraries/System.Private.CoreLib/src/System/Random.Xoshiro256StarStarImpl.cs

```diff
@@ -200,9 +201,8 @@ public override unsafe void NextBytes(Span<byte> buffer)
 
                 while (buffer.Length >= sizeof(ulong))
                 {
-                    Unsafe.WriteUnaligned(
-                        ref MemoryMarshal.GetReference(buffer),
-                        BitOperations.RotateLeft(s1 * 5, 7) * 9);
+                    BinaryPrimitives.WriteUInt64LittleEndian(
+                        buffer, BitOperations.RotateLeft(s1 * 5, 7) * 9);
```

Comment:
Why does endianness matter in this implementation?

Reply:
Yes, there are tests that verify a particular result.   If this doesn't actually matter, I guess we can instead skip the test?

Reply:
I don't think it actually matters; we make no guarantees as the algorithm that's used.  Let's just skip the relevant parts of the test. Thanks.

Reply:
OK, will do.